includes/class-wcpdf-admin.php

```diff
@@ -347,12 +347,16 @@ public function request_query_sort_by_column( $query_vars ) {
 	/**
 	 * Add the meta boxes on the single order page
 	 */
-	public function add_meta_boxes() {
+	public function add_meta_boxes( $wc_screen, $wc_order ) {
```

Comment:
I guess we can skip $wc_order variable as it is not used in function.

Reply:
I think we should keep for future reference, if we need to use it later, but I will add instructions to the comment above.